spec/v0.4.1-draft/spec.md

```diff
@@ -332,17 +328,40 @@ NOTE: JSON numbers allow integer and floating-point values, so does OCIF.
   - Resource can be empty, in which case a node is acting as a transform for other nodes.
   - z-ordering: The resource is to be rendered behind the node. For example, if the node has a rectangular border (or [oval](#oval)), that border would be rendered in front of the resource.
 
-- **rotation**: The 2D rotation of the node in degrees. The rotation center is the positioned point, i.e., top-left. The z-axis is not modified.
+- **resource-fit**: Given a node with dimensions 100 (height) x 200 (width) and a bitmap image (e.g., a .png) with a size of 1000 x 1000.
+  How should this image be displayed? We re-use some options from CSS ([object-fit](https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit) property):
 
-- **scale**: Allows to re-scale a given node.
-  NOTE: This is particularly useful if the [parent-child](extensions.md#parent-child-relation) relation extension applies to the node and child nodes need a consistent scale transform.
+  - `none`: All pixels are displayed in the available space unscaled. The example would be cropped down to the 100 x 200 area top-left. No auto-centering.
+  - `keep-width`: Scaled by keeping the aspect ratio, so that the image width matches the item width. This results in the image being displayed at a scale of `0.5`, so that it is 200 px wide and 200 px heigh. The top half of the image is visible.
+  - `keep-height`: Scaled by keeping the aspect ratio, so that the image height matches the item height. This results in the image being displayed at a scale of `0.1`, so that it is 100 px high and 100 px wide. The image is now fully visible, but there are boxes of empty space left and right of the image.
```

Comment:
```suggestion
  - `keepWidth`: Scaled by keeping the aspect ratio, so that the image width matches the item width. This results in the image being displayed at a scale of `0.5`, so that it is 200 px wide and 200 px heigh. The top half of the image is visible.
  - `keepHeight`: Scaled by keeping the aspect ratio, so that the image height matches the item height. This results in the image being displayed at a scale of `0.1`, so that it is 100 px high and 100 px wide. The image is now fully visible, but there are boxes of empty space left and right of the image.
```

Reply:
keepHeight -> containY
keepWidth -> containX